stock_predictor/service.py

```diff
@@ -56,13 +64,12 @@ def predict_all(date=None):
     Predict for all stocks in given date. If no date is given, predict for all the dates.
 
     Args:
-        date: The date to predict.
-        force_update: Never skip any stocks if set to `True`.
+        date: The date to predict.        
     """
     qlib.init(provider_uri='~/.qlib/qlib_data/cn_data')
     database = TinyDB(STOCK_DATABASE)
     if date is None:
-        date = datetime.date.today().strftime('%Y-%m-%d')
+        date = '2022-01-01'
```

Comment:
We should make start_date and end_date different if date is None.
Let `start_date = '2022-01-01'` and `end_date = datetime.date.today().strftime('%Y-%m-%d')`.